### DISCUSSION TITLE
Hava durumu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#239] [Zeka Sorusu]
Irem, Mahsun ve Gözde arasında geçen haftanın hava durumu ile ilgili şu konuşmalar geçer.
1. kişi: Salı günü yağmurluydu.
2. kişi: Çarşamba günü karlıydı.
3. kişi: Cuma günü yağmurluydu.
2. kişi: Salı günü güneşliydi.
3. kişi: Perşembe yağmurluydu.
1. kişi: Cuma karlıydı.
2. kişi: Evet cuma karlıydı.
1. kişi: Çarşamba yağmurluydu.
3. kişi: Pazartesi güneşliydi.
2. kişi: Cumartesi ve Pazar karlıydı.
Bu üç kişiden Irem hep doğru, Mahsun hep yalan söylemektedir.
Gözde ise Iremden hemen sonraki konuşmalarında doğru söylemektedir. Gün içinde hava değişmediğine göre geçen haftanın hava durumunu bulunuz.

**Comment by Adaisland** [comment]
Pazartesi:tek bilgimiz var (güneşli)
Salı:perşembe yağmurlu olduğundan gözde söylemiş dersek 4. Kişiyi irem kabul edip güneşli diyebiliriz
Çarşamba:(yağmurlu)9.kişinin pazartesi hakında söyledinin doğru olduğunu bildiğimiz için bu kişi yi gözde olarak alırsak irem söyledi diyebiliriz
Perşembe:tek bilgiye sahibiz irem söyledi sayıyoruz yağmurlu
Cuma:5. Kişi irem 6. Kişi gözdeyse karşıdır muhsinse ve 7. Kişi iremsede hava karlıdır
Pazar ve Cumartesi: Bu iki gün için tek bilgiye sahip olduğumuz için ve hava durumu listesi oluştuğuna göre irem söyledi sayıyoruz (karlı)

Pazartesi:Güneşli
Salı:Güneşli
Çarşamba:yağmurlu
Perşembe:yağmurlu
Cuma:karlı
Cumartesi:karlı
Pazar:karlı
